core/trino-main/src/main/java/io/trino/FeaturesConfig.java

```diff
@@ -514,6 +516,18 @@ public FeaturesConfig setFaultTolerantExecutionExchangeEncryptionEnabled(boolean
         return this;
     }
 
+    public boolean isStatementRedactingEnabled()
+    {
+        return statementRedactingEnabled;
+    }
+
+    @Config("statement-redacting-enabled")
```

Comment:
@mosabua for suggestions about config naming. 😄

Reply:
I'm not sure we want an option to disable this.  Maybe as a temporary kill switch, but we should remove this as soon as we are happy with this feature

Reply:
Agreed, we can prefix with `experimental.` in that case like we have done in past to clarify this. Or maybe `deprecated.` from the beginning.